src/applications/mhv-medical-records/containers/App.jsx

```diff
@@ -290,7 +290,7 @@ const App = ({ children }) => {
                 )}
                 <div className="vads-l-grid-container vads-u-padding-x--0 vads-u-margin-x--0 vads-u-flex--fill">
                   <div className="vads-l-row">
-                    <div className="vads-l-col">{children}</div>
+                    <div className="vads-l-col--8">{children}</div>
```

Comment:
This `--8` is causing issues with responsive layout. What was the reason behind this change?

<img width="251" alt="image" src="https://github.com/user-attachments/assets/ef9fa465-d593-4fac-b454-279253f9ca3a">

Reply:
I saw instructions that said to make the column "8 columns wide" and I thought that's what they meant. I can't find i on the Figma anymore so I guess somebody changed their mind imma change it back.